background/get-addon-settings.js

```diff
@@ -592,6 +592,13 @@ chrome.storage.sync.get([...ADDON_SETTINGS_KEYS, "addonsEnabled"], (storageItems
             addonSettings["comment-color"] = "#FEF49C";
           }
         }
+        if (addonId === "forum-quote-code-beautifier") {
+          if (settings.bordercolor === "#28A5DA") {
+            // Transistion v1.33 to v1.34
+            settings.bordercolor = "#855CD6";
+            madeAnyChanges = madeChangesToAddon = true;
+          }
+        }
```

Comment:
I think this will continuously force the user to return to the previous setting if they come back to the old color.